modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -0,0 +1,139 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-host-dual-network-interfaces-in-the-install-config-yaml-file_{context}"]
+= Optional: Configuring host network interfaces for dual port NIC
+
+[IMPORTANT]
+====
+Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+
+.Prequisites
+
+* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
+* Install the NMState CLI (`nmstate`).
+
+[NOTE]
+====
+Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
+====
+
+.Procedure
+
+. Use the `networkConfig` configuration setting by adding the NMState configuration to hosts within the `install-config.yaml` file:
```

Comment:
This didn't make sense to me and I had to re-read it a few times to see what was meant. 

I think this could be reworded to be clearer. What about

```suggestion
. Add the NMState configuration to the `networkConfig` field to hosts within the `install-config.yaml` file:
```